apps/fluid-dev/src/pages/button-group/button-group-page.component.html

```diff
@@ -2,10 +2,10 @@
   (checkedChange)="handleChange($event)"
   [disabled]="disableall"
 >
-  <fluid-button-group-item name="btngrp">
+  <fluid-button-group-item name="btngrp" [disabled]="disable">
```

Comment:
NIT: Personally I feel this is  bit off :). I think this should be "disabled", and "disableAll".

Reply:
@bardosmisi One disabled property is on the wrapper so I thought maybe users would understand that this disables all inside items. I agree that `disableAll` is more clear and easy to understand